fishtank/src/cluster.ts

```diff
@@ -36,4 +36,19 @@ export class Cluster {
     })
     return new Node(this, name)
   }
+
+  async teardown(): Promise<void> {
```

Comment:
is this teardown a network or tear down the cluster?

Reply:
The whole cluster.